src/Providers/ActionProvider.php

```diff
@@ -33,5 +35,10 @@ public function register()
         });
 
         $this->getContainer()->share(Actions\Chore\ListChores::class);
```

Comment:
I forget, did we decide on `$this->getContainer()` or `$this->container`? Either way, we should be consistent.